### PR TITLE
Homeappdevbranch

### PR DESCRIPTION
**Pull request by arun-msft**

### CODE REVIEW

File: HOMEapp/README.md [hunk 1]
```diff
@@ -181,10 +181,10 @@ Deployment
 
 -   Specify who can use the application, as follows:
 
--  Accounts in any organizational directory (Any Azure Ad directory --Multitenant) and personal Microsoft accounts.
+-   Accounts in any organizational directory (Any Azure Ad directory --Multitenant) and personal Microsoft accounts.
```
Comment (aamini-ms) [inline HOMEapp/README.md:184]: What's the diff. between these 2?

File: HOMEapp/README.md [hunk 2]
```diff
@@ -181,10 +181,10 @@ Deployment
...
 
--   Under Redirect URI select Public client/native (mobile & desktop).Then enter the destination URI that will be accepted when returning authentication responses (tokens)
-    after successfully authenticating users. 
+-   Under Authentication, click on Add a platform and select Web.Then enter the redirect URI that will be accepted when returning authentication responses (tokens)
+    after successfully authenticating users. Make sure the redirect URI follows the following format, https://%appDomain%.azurewebsites.net/End. Redirect URI can be set post     deployment also, if the appDomain is not available yet.
```
Comment (aamini-ms) [inline HOMEapp/README.md:188]: Under Authentication, click on Add a platform and select Web. Then enter the redirect URI that will be accepted when returning authentication responses (tokens).
              - After successfully authenticating users, make sure the redirect URI follows the following forma:, 
                 https://%appDomain%.azurewebsites.net/End. 
              - Note, redirect URI can be set post deployment also, if the appDomain is not available yet.

File: HOMEapp/Brandhome/Manifest/manifest.json [hunk 3]
```diff
@@ -8,7 +8,8 @@
     "name": "Microsoft",
     "websiteUrl": "https://www.microsoft.com",
     "privacyUrl": "https://www.microsoft.com/privacy",
-    "termsOfUseUrl": "https://www.microsoft.com/termsofuse"
+    "termsOfUseUrl": "https://www.microsoft.com/termsofuse",
+    "mpnId": "{your mpnID}"
   },
```
Comment (aamini-ms) [inline HOMEapp/Brandhome/Manifest/manifest.json:13]: Add a comment here, <-- Microsoft Partners must include their MPNID in the App Manifest when they build the App. It is being used for telemetry and usage reporting purposes -->

Comment (arun-msft) [reply HOMEapp/Brandhome/Manifest/manifest.json:13]: Updated

File: HOMEapp/README.md (hunk 2 above)
Comment (arun-msft) [reply HOMEapp/README.md:187]: Updated

File: HOMEapp/README.md (hunk 1 above)
Comment (arun-msft) [reply HOMEapp/README.md:184]: updated

**Pull request merged**